Gemfile.lock

```diff
@@ -107,9 +109,9 @@ PLATFORMS
 
 DEPENDENCIES
   active_record_doctor!
-  rails (~> 4.2)
+  rails (>= 4.2, < 5.0)
```

Comment:
It seems we need to rerun `bundle install`. Could you confirm, @syndbg? Also see https://travis-ci.org/gregnavis/active_record_doctor/builds/161303757